tests/profiling/test_profiler.py

```diff
@@ -258,7 +258,7 @@ def test_gevent_warning(monkeypatch):
     )
     assert subp.wait() == 0
     assert subp.stdout.read() == b""
-    assert b"RuntimeWarning: Starting the profiler before using gevent" in subp.stderr.read()
+    assert b"RuntimeWarning: Loading ddtrace before using gevent monkey patching" in subp.stderr.read()
```

Comment:
should this test case be moved outside of profiling tests?

Reply:
I thought about it but was not sure where to put it. We don't have a place where we run the internal/common stuff. It's all in `tracer`. Should I move it here?

Reply:
yeah, that is probably where I would put it `tests/tracer` is where a lot of the internal/common tests go.